packages/flutter_tools/lib/src/commands/generate_localizations.dart

```diff
@@ -207,6 +207,10 @@ class GenerateLocalizationsCommand extends FlutterCommand {
             'and "}" is treated as a string if it does not close any previous "{" '
             'that is treated as a special character.',
     );
+    argParser.addFlag(
+      'use-named-argument',
```

Comment:
About the name of this new flag:

Should `use-named-argument` be `use-named-arguments` since any given localization method may have multiple parameters?

Less importantly, I think a better name for this might be `--use-named-parameters`, since the flag (at least in my mind) is referring to the names listed in the method definitions, not the values being passed (the arguments).

Reply:
@andrewkolos I have done changing to --use-named-parameters.